plugins/SitesManager/Controller.php

```diff
@@ -146,6 +148,34 @@ public function siteWithoutData()
             'idSite' => $this->idSite
         ), $viewType = 'basic');
 
+        return $this->renderTemplateAs('siteWithoutData', array(
+            'siteName'      => $this->site->getName(),
+            'idSite'        => $this->idSite,
+            'piwikUrl'      => $piwikUrl,
+            'emailBody'     => $emailContent,
+        ), $viewType = 'basic');
+    }
+
+    public function siteWithoutDataTabs() {
+        try {
+            $response = Http::sendHttpRequest($this->site->getMainUrl(), 5, null, null, 0, false, false, true);
+        } catch (Exception $e) {
+            $response = false;
+        }
+        $guesser = new GtmSiteTypeGuesser();
+        $siteType = $guesser->guessSiteTypeFromResponse($response);
+        $gtmUsed = $guesser->guessGtmFromResponse($response);
+        $instructionUrl = SitesManager::getInstructionUrlBySiteType($siteType);
```

Comment:
Can we cache `$siteType` / `$gtmUsed` so we don't send an HTTP request every time this page is viewed? Here's one example of how the cache is used: https://github.com/matomo-org/matomo/blob/fd14a155269038f6fe12ce31db20f8ebffaf00c5/plugins/RssWidget/RssRenderer.php#L52